client/main.go

```diff
@@ -90,8 +71,23 @@ func NewClientConn(opt *ConnOptions) (*grpc.ClientConn, error) {
 		return nil, errors.Wrap(err, "parse cert")
```

Comment:
I know we didn't create this issue, but I just realized that `err` here is `nil`.

An improvement would be decoding the certificate's PEM and parsing the resulting ASN.1 DER using the `x509` library. That, or creating a new error that says something like `failed to parse cert - cert pool append certs from pem failed`.

Reply:
I'd agree, but I'm actually looking to drop the `/cert` completely so this will go away anyway.